core/src/main/java/com/linecorp/armeria/server/docs/EnumValueInfo.java

```diff
@@ -50,23 +50,23 @@ public EnumValueInfo(String name) {
     /**
      * Creates a new instance.
      *
-     * @param name the name of the enum value
-     * @param literalValue literal value (for example, int value) for the enum value
+     * @param name the integer value of the enum value
+     * @param intValue the current value of the enum value
```

Comment:
name -> the name of the enum value
intValue -> the integer value of the enum value